e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -57,4 +59,22 @@ describe("scenarios > dashboard tabs", () => {
       cy.findByText("Orders").should("be.visible");
     });
   });
+
+  it("should update slug in url after creating a new tab and saving", () => {
+    visitDashboardAndCreateTab({ dashboardId: 1 });
+    cy.url().should("include", "2-page-2");
```

Comment:
Wdyt about adding this assertion to `should only display cards on the selected tab` without adding a new test?